lib/helpers/helpers-html.js

```diff
@@ -157,6 +157,16 @@ module.exports.register = function (Handlebars, opts) {
       }
       figure += '</figure>';
       return new Handlebars.SafeString(figure);
+    },
+
+    /**
+     * {{#striptags}}{{{data}}}{{{//striptags}}}
+     * Remove all HTML tags from the nested context
+     * @param  {[type]} options [description]
+     * @return {[type]}         Text with HTML tags removed
+     */
+    striptags: function (options) {
```

Comment:
:+1:  to add a little more to what @doowb mentioned (b/c it took me forever to remember these), when you do `console.log()` on those objects, if you see `hash`, it's really the `options` object that is created by Handlebars, with additional properties that handlebars creates, like `root`. But context is the actual data object that is being built-up from various sources to pass to your templates. Both of these objects can be merged inside the helper function. Hope that helps (or maybe it made it worse lol)